src/app/shared/pipes/time-duration/timestamp.ts

```diff
@@ -37,6 +37,19 @@ export class Timestamp {
     );
   }
 
+  parseToMilliseconds(value: ParseTimestampResult): number {
```

Comment:
```suggestion
    parseToMilliseconds(value: ParseTimestampResult): number {
    const seconds = this.parseToSeconds(value);
    return seconds ? seconds * 1000 : null;
  }
```

Would be good to add some tests as well.